modules/gapi/src/backends/fluid/gfluidimgproc.cpp

```diff
@@ -1952,10 +1954,18 @@ struct MapperUnit {
     I index0, index1;
 };
 
-inline static uint8_t calc(short alpha0, uint8_t src0, short alpha1, uint8_t src1) {
+CV_ALWAYS_INLINE uint8_t calc(short alpha0, uint8_t src0, short alpha1, uint8_t src1)
+{
     constexpr static const int half = 1 << 14;
     return (src0 * alpha0 + src1 * alpha1 + half) >> 15;
```

Comment:
IMHO:
is it blend operation?  because from `calc` description it hard to understand :)

Reply:
This function does calculation that convert fixed point result values to usual view.

Reply:
IMHO: it would be good to reflect it in function name

Reply:
Applied.